### PR TITLE
intro

### PR DESCRIPTION
**Pull request by imparang**
Use description list tag for introducing my self.

### CODE REVIEW

File: introduce.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>My First HTML Page with git</title>
+	</head> 
+	<body>
+		<h1>안녕하세요. 정재윤입니다. </h1>
+		<dl>
```
Comment (lazy-sky) [inline introduce.html:10]: I think it's better to separate the contents by category.

File: introduce.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>My First HTML Page with git</title>
+	</head> 
+	<body>
+		<h1>안녕하세요. 정재윤입니다. </h1>
```
Comment (lazy-sky) [inline introduce.html:9]: could add more details with p tag?

**Review by lazy-sky — CHANGES_REQUESTED** [review]

File: introduce.html (hunk 1 above)
Comment (imparang) [reply introduce.html:10]: Okay. I will modify this statements.

File: introduce.html (hunk 2 above)
Comment (imparang) [reply introduce.html:9]: No problem.

**Review by lazy-sky — APPROVED** [review]
Excellent work!

**Pull request merged**